ospf6d/ospf6_top.h

```diff
@@ -230,7 +230,7 @@ struct ospf6 {
 
 #define OSPF6_EXTL_AGGR_DEFAULT_DELAY 5
 	/* For ASBR summary delay timer */
-	int aggr_delay_interval;
+	unsigned int aggr_delay_interval;
```

Comment:
uint32_t, but I see this is even uint16_t (according to the values it can be).

Reply:
Yes, uint16_t is ok. But i don’t want to change a lot. And i just refer to ospf4, which uses the “unsigned int”. Need change?

Reply:
I think if we are already here, we can change it?

Reply:
Okay.
Thanks for review, it is done with last push.